<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cMainWindowClass</class>
 <widget class="QMainWindow" name="cMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Combattants</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listCombattants"/>
        </item>
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
        <item>
         <widget class="QTreeView" name="treeView_2"/>
        </item>
        <item>
         <widget class="QTreeView" name="treeView_3"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="cCombattantView" name="CombattantView" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="FormView" name="formView" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Targets</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listTargets"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="cTargetView" name="TargetView" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>cCombattantView</class>
   <extends>QWidget</extends>
   <header>combattantview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>cTargetView</class>
   <extends>QWidget</extends>
   <header>targetview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormView</class>
   <extends>QWidget</extends>
   <header>FormView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="cMainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
